daemon/cmd/daemon_main.go

```diff
@@ -111,21 +111,10 @@ var (
 				os.Exit(0)
 			}
 
-			// Open socket for using gops to get stacktraces of the agent.
-			addr := fmt.Sprintf("127.0.0.1:%d", viper.GetInt(option.GopsPort))
-			addrField := logrus.Fields{"address": addr}
-			if err := gops.Listen(gops.Options{
-				Addr:                   addr,
-				ReuseSocketAddrAndPort: true,
-			}); err != nil {
-				log.WithError(err).WithFields(addrField).Fatal("Cannot start gops server")
-			}
-			log.WithFields(addrField).Info("Started gops server")
```

Comment:
It looks like gops agent integration is removed entirely by this PR. Is this intentional? I think we preferably want to keep gops for debugging.

Reply:
Oops, definitely was not intentional! Rebasing accident. I'll fix asap.